doc/manual/metaprogramming.rst

```diff
@@ -620,11 +629,14 @@ Compare:
     Expr
       head: Symbol string
       args: Array{Any}((5,))
-        1: String "a ("
+        1: String
+          data: Array{UInt8}((3,)) UInt8[0x61,0x20,0x28]
```

Comment:
bisected to 3405d263194 so I guess it was intentional - the new output isn't quite as nice, but I guess it says "`dump` should not be overloaded" so I guess we're stuck with it